tests/integration-tests/tests/cloudwatch_logging/cloudwatch_logging_boto3_utils.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+
+import utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_log_groups():
+    """Get list of log groups."""
+    logs_client = boto3.client("logs")
+    log_groups = logs_client.describe_log_groups().get("logGroups")
+    LOGGER.debug("Log groups: {0}\n".format(json.dumps(log_groups, indent=4)))
```

Comment:
What about defining an utility function to format the json?
I see we are performing the `json.dumps(..., indent=4)` in multiple places.

Reply:
Done.